### PR TITLE
Fix various errors in logs

### PR DESCRIPTION
**Pull request by emosbaugh**

### CODE REVIEW

File: conf/graphite/carbon.conf [hunk 1]
```diff
@@ -30,8 +30,24 @@
...
+# whisper - Fixed-size database, similar in design and purpose to RRD. This is
+# the default storage backend for carbon and the most rigorously tested.
+#
+# ceres - Experimental alternative database that supports storing data in sparse
+# files of arbitrary fixed-size resolutions.
+DATABASE = whisper
+
+# Enable daily log rotation. If disabled, a new file will be opened whenever the log file path no
+# longer exists (i.e. it is removed or renamed)
+# Hopefully fixes FileNotFoundError: [Errno 2] No such file or directory: '/opt/graphite/storage/log/webapp/info.log'
```
Comment (emosbaugh) [inline conf/graphite/carbon.conf:46]: Fix

File: conf/graphite/carbon.conf [hunk 2]
```diff
@@ -170,19 +304,26 @@ WHISPER_AUTOFLUSH = False
 # Example: store everything
 # BIND_PATTERNS = #
 
+# URL of graphite-web instance, this is used to add incoming series to the tag database
+# Fixes error [tagdb] Error tagging xxx: Error requesting http://127.0.0.1:80/tags/tagMultiSeries: ... [Errno 111] Connection refused
```
Comment (emosbaugh) [inline conf/graphite/carbon.conf:308]: fix

File: Dockerfile [hunk 3]
```diff
@@ -77,7 +77,11 @@ RUN mkdir -p /var/log/graphite/ \
   && PYTHONPATH=/opt/graphite/webapp /opt/graphite/bin/django-admin.py collectstatic --noinput --settings=graphite.settings \
   && PYTHONPATH=/opt/graphite/webapp /opt/graphite/bin/django-admin.py migrate --noinput --settings=graphite.settings --run-syncdb
 
-RUN chmod -R a+rwx /opt/graphite/storage
+RUN chmod -R a+rwx \
+  /opt/graphite/storage \
+  # Fixes error Unable to write to plugin cache
```
Comment (emosbaugh) [inline Dockerfile:82]: fix

**Comment by emosbaugh** [comment]
the rest of this is just copying over the latest conf from upstream

**Review by divolgin — APPROVED** [review]

**Pull request merged**